Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/SocketerNetworkingService.cs

```diff
@@ -0,0 +1,188 @@
+﻿using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing;
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// Networking class based on Socketer's <see cref="Experimental.Socketer.NetworkConnectionManager"/>
+    /// </summary>
+    public class SocketerNetworkingService : MonoBehaviour,
+        IMatchMakingService,
+        IPlayerService,
+        INetworkingService
+    {
+        /// <summary>
+        /// TCPConnectionManager used for communicating with other devices
+        /// </summary>
+        [Tooltip("TCPConnectionManager used for communicating with other devices")]
+        [SerializeField]
+        protected TCPConnectionManager net = null;
+
+        /// <summary>
+        /// IPAddress for the user (HoloLens & HoloLens 2)
+        /// </summary>
+        [Tooltip("IPAddress for the user (HoloLens & HoloLens 2)")]
+        [SerializeField]
+        protected string userAddress = "127.0.0.1";
+
+        /// <summary>
+        /// Port for the user (HoloLens & HoloLens 2)
+        /// </summary>
+        [Tooltip("Port for the user (HoloLens & HoloLens 2)")]
+        [SerializeField]
+        protected int userPort = 7997;
+
+        /// <summary>
+        /// Timeout interval for establishing connections
+        /// </summary>
+        [Tooltip("Timeout interval for establishing connections")]
+        [SerializeField]
+        protected TimeSpan timeoutInterval = TimeSpan.Zero;
+
+        private bool runAsUser = false;
+        private Dictionary<string, string> playerIds = new Dictionary<string, string>();
```

Comment:
>private Dictionary<string, string> playerIds = new Dictionary<string, string>() [](start = 8, length = 79)

Nit: Can you add a comment about what this dictionary is? i.e. seeing a string to string map and I wonder what it is.